src/aws/establishIAMRole.js

```diff
@@ -7,7 +8,8 @@ const establishIAMRole = async (roleName) => {
   } catch(err) {
     console.log(err.message);
     console.log(`Creating a new role called ${roleName}...`);
-    return await iam.createRole(generateRoleParams(roleName)).promise();
+    return await iam.createRole(generateRoleParams(roleName)).promise()
+    .then(() => sleep(7000));
```

Comment:
I like that the `sleep` is in this file. If the policies are attached after the sleep period and immediately before the creating of functions, does that give us the same error? Or does it just depend on when the role was created, rather than when the policies are attached?

Reply:
It does not seem to matter when the policies are attached, only when the role was created. It may be better to sleep before the attaching the policies..if we do that then we may not need to use the retryAsync with attachPolicies. What are your thoughts?

Reply:
That's a good observation. I'm in favor of abolishing the retry for attaching policies.

Reply:
OK, sounds good. I'll remove the logic, test it once more, and then submit the PR.